component/appcat_apiserver.jsonnet

```diff
@@ -207,7 +208,7 @@ local apiCertificate = {
   },
 };
 
-if apiserverParams.enabled == true then {
+if apiserverParams.enabled && vars.isSingleOrControlPlaneCluster then {
```

Comment:
The api server does not run in service cluster?

Reply:
nope. Why should it?